### ISSUE TITLE
Completing website

### ISSUE THREAD

**Issue by moorlag** [3PMAP/3dprints#4]
Possible to publish documentation in a simple site?

**Comment by jelle641** [comment]
Yes I started on programming one for our documentation. Maybe we can also take a look at github pages? Maybe that is a easier solution than programming one on our own

**Comment by moorlag** [comment]
❤️GitHub Pages

**Comment by jelle641** [comment]
I already coded a simple website https://3pmap.github.io/website/  (repo website inside the organisation) But I think that github pages will work 1000 times better

**Comment by yochem** [comment]
Created Jekyll + Github pages [branch](https://github.com/3PMAP/website/tree/jekyll) in website repo

**Comment by jelle641** [comment]
Je mag hem meteen naar master pushen als je wilt, hoeft van mij niet in een branch @yochem

**Comment by jelle641** [comment]
The website is made, now we just have to add all the aditional information
